dev/tests/functional/standalone_bootstrap.php

```diff
@@ -49,7 +49,10 @@
 
     defined('DEFAULT_TIMEZONE') || define('DEFAULT_TIMEZONE', 'America/Los_Angeles');
     $env->setEnvironmentVariable('DEFAULT_TIMEZONE', DEFAULT_TIMEZONE);
-    
+
+    defined('WAIT_TIMEOUT') || define('WAIT_TIMEOUT', 10);
+    $env->setEnvironmentVariable('WAIT_TIMEOUT', 10);
```

Comment:
Is 10 seconds sufficient for page load? especially when page is not cached.

Reply:
Honestly I don't think so. We can change this default to 30.
I changed it to 10 as that's what it was in ActionObject, that may have been the wrong call.